src/app/json-editor/json-editor.component.ts

```diff
@@ -89,8 +97,8 @@ export class JsonEditorComponent implements OnInit {
           const resSTR = JSON.stringify(json);
           this.jsonSource = JSON.parse(resSTR);
           this.jsonSource = JSON.stringify(JSON.parse(resSTR), null, '    ');
-        } catch (ex) {
-          alert('exception when trying to parse json = ' + ex);
+        } catch (e) {
+          alert('Exception when trying to parse json');
```

Comment:
remove javascript alert and show modal dialog

Reply:
Removed Javascript alert.